samples/databases/wide-world-importers/wwi-dw-ssdt/README.md

```diff
@@ -0,0 +1,80 @@
+# WideWorldImportersDW OLAP Database
+
+The Visual Studio SQL Server Data Tools project in this folder is used to construct the WideWorldImportersDW database from scratch on SQL Server or Azure SQL Database. It is possible to vary the data size.
```

Comment:
Would this VSDT project be backward compatible? If not we should specify starting that "version of SQL Server or Azure SQL Database"